core/src/bake-variable.ts

```diff
@@ -12,7 +12,9 @@ export class BakeVariable {
     _compiled: Function | undefined | null
 
     public get Code(): any {
-        return this._value || ""
+        if (this._value == null || this._value == undefined)
```

Comment:
why this change? the or operator returns false if the value is null or undefined, so this change should result in the same outcome as the more standard or op check.

Reply:
The previous code was also returning "" when this._value was set to boolean false

Reply:
were you solving an issue where a value of false or 0 (non string) was getting converted to empty str?

Reply:
Yes I came across the issue again when setting the isEnabled values to false in the stockalerts.json.  The ARM templates for the alerts were then being rejected by Azure because it said I was passing in an invalid value.  The debug output showed it was passing "" and I traced the issue back to that method.